changelog/5115.bugfix.rst

```diff
@@ -0,0 +1 @@
+Capture warnings during ``pytest_configure``.
```

Comment:
```suggestion
Warnings issued during ``pytest_configure`` are explicitly not treated as errors, even if configured as such, because it otherwise completely breaks pytest.
```